dissect/target/target.py

```diff
@@ -639,7 +632,7 @@ def add_plugin(
         if not isinstance(p, plugin.Plugin):
             raise PluginError(f"Not a subclass of Plugin: {p}")
 
-        if check_compatible:
+        if check_compatible and not self.is_direct:
```

Comment:
I also tried this solution, but ran into an issue that some plugins tried to load which should not load (something with fortis IIRC).

The tests seem to pass, but I still wonder if this is safe?

Reply:
The root cause was that DefaultOS didn´t have a date time & locale plugin. This caused the plugin system to try date time & locale plugins of other OS'es, like FortiOS (this is the behavior only when the DefaultOS is selected). During the Ruff refactor, a default date time & locale plugin has been added.

In principle, turning off compatibility checks when loading a plugin is safe; we should a default implementation for date  time & local plugins.

In conclusion, I prefer this over  the previous implementation, because when single file / direct mode is engaged, you typically don´t have any meaningful checks.